netmap/netmap_contract.go

```diff
@@ -601,6 +610,37 @@ func ListConfig() []record {
 	return config
 }
 
+// SubscribeForNewEpoch registers passed contract as a NewEpoch event
+// subscriber. Such a contract must have a `NewEpoch` method with a single
+// numeric parameter. Transactions that call SubscribeForNewEpoch must be
+// witnessed by the Alphabet.
+// Produces `NewEpochSubscription` notification event with a just
+// registered recipient in a success case.
+func SubscribeForNewEpoch(contract interop.Hash160) {
+	common.CheckAlphabetWitness(common.AlphabetAddress())
+
+	callingContract := runtime.GetExecutingScriptHash()
```

Comment:
And here you have `contract`, executing script hash is netmap.

Reply:
wow, fixed